src/sentry/static/sentry/app/components/organizations/headerItem.tsx

```diff
@@ -189,7 +189,7 @@ const StyledChevron = styled(IconChevron, {shouldForwardProp: isPropValid})<{
   color: ${getColor};
 `;
 
-const SettingsIconLink = styled(Link)`
+export const SettingsIconLink = styled(Link)`
```

Comment:
We should move this into a new file if it's being imported outside of `HeaderItem`

Reply:
Good point. I'll try to find a home for it.